python/boat_controller_node.py

```diff
@@ -179,11 +191,19 @@ def main():
     rospy.Subscriber("/min_voltage", min_voltage, minVoltageCallBack)
     rospy.Subscriber('/lowWindConditions', Bool, lowWindCallBack)
 
-    if DISABLE_LOW_POWER:
-        rospy.logwarn("Low power mode is DISABLED! If you don't want this, change DISABLE_LOW_POWER to False.\n")
-    rospy.loginfo("Boat controller started successfully. Waiting on sensor data...\n")
-
-    rospy.spin()
+    if (FIXED_CTRL_STATE is not None) and (DISABLE_LOW_POWER):
+        rospy.logerr("Cannot disable low power and fix the control state at the same time!")
+        rospy.signal_shutdown("Low power mode is disabled and the control state is fixed.")
```

Comment:
Rather than having certain combinations of variables that cause your boat_controller to shut down, why don't you eliminate that possibility altogether? For example, just have one variable with 5 states:
1. Everything
2. Disable low power
3. Jibe
4. Tack
5. Low power